scss/_variables.scss

```diff
@@ -1647,3 +1701,5 @@ $kbd-bg:                            var(--#{$prefix}body-color) !default;
 $nested-kbd-font-weight:            null !default; // Deprecated in v5.2.0, removing in v6
 
 $pre-color:                         null !default;
+
+@import "variables-dark";
```

Comment:
I'm not sure about this import here. It shouldn't always be imported depending on if the dark mode is enabled or not. Not entirely sure but it should rather be done "outside" voluntarily by the users. But it means updating our doc regarding the Sass customization (see other comment in this PR).
I'm saying that because they could have their "variables-high-contrast", "variables-whatever", and so on.